tomlkit/exceptions.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import annotations
```

Comment:
This only works on 3.7+ IIRC?

Reply:
Oh, right. I knew I forgot about something. I guess I will need to do some reverting.

Reply:
Reverted.